contents/blog/give-back-friday-2021.md

```diff
@@ -0,0 +1,73 @@
+
+## Who gets the money? 
+
+We’re donating to the following projects, all of which are either used extensively in our website or product, or have been suggested for their daily usefulness to our team outside of work. 
+
+| **Project**                                                                | **What is it?**                                                         | **Why we're supporting it**                                                                     |
+|--------------------------------------------------------------------------|-----------------------------------------------------------------------|------------------------------------------------------------------------------------------------|
+| [Rrweb](https://github.com/rrweb-io/rrweb)                               | An open-source web session replay library.                            | Rrweb powers [our session recording features](https://posthog.com/docs/user-guides/recordings) |
+| [Graphile/Worker](https://github.com/graphile/worker)                    | A high performance Node.js/PostgreSQL job queue.                      | Graphile/Worker powers logic for our plugin server.                                            |
+| [Django REST framework](https://github.com/encode/django-rest-framework) | Open-source web APIs for Django.                                      | This powers the PostHog REST APIs.                                                             |
+| [KeePass](https://keepass.info/donate.html)                              | An open-source multi-platform password manager.                       | Keepass is used regularly by our team and was suggested by Joe Martin.                         |
+| [Piscina](https://github.com/piscinajs/piscina)                          | A fast, efficient Node.js Worker Thread Pool implementation.          | We run a fork of Piscina which powers activities across PostHog.                               |
+| [All-contributors](https://github.com/all-contributors/all-contributors) | A tool to recognize community contributions to open-source projects.  | We run a form of All-contributors to power our contributions bot.                              |
+| [Alex](https://github.com/all-contributors/all-contributors)             | A tool to detect insensitive or inconsiderate writing.                | Alex is used by our team and was suggested by Paul D'Ambra.                                    |
+| [Ant-design](https://github.com/all-contributors/all-contributors)       | An open-source UI design language and React UI library.               | Ant-design is used heavily across both our website and product.                                |
```

Comment:
```suggestion
| [Ant-design](https://github.com/ant-design/ant-design)       | An open-source UI design language and React UI library.               | Ant-design is used heavily across both our website and product.                                |
```

Reply:
I'm not sure it's rational to give to ant design - I think (but please check) this is owned by Alibaba who definitely have a lot of money, so I think we'll help people out way more for each $ given if we focus this on non commercially-owned projects

Reply:
Given feedback in [the issue](https://github.com/PostHog/company-internal/issues/480#issuecomment-977271977), let's remove ant design and not replace with anything else?

Reply:
Will remove. Clearly some crossed wires in the suggestions there!